README.md

```diff
@@ -44,6 +44,7 @@ var auth0GuardianJS = require('auth0-guardian-js')({
 	globalTrackingId: "{{ globalTrackingId }}"
 });
 ```
+Use `requestToken` or `ticket` depends on the authentication method. Ticket correponds to a previous generated  _enrollment ticket_. 
```

Comment:
Basically, I'd say that both `requestToken` and `ticket` needs more explaination. Like, how to get one, pointer to the documentation of how to get a ticket for example.  We don't have any documented way to get a requestToken yet though...